runtime/darwinia/src/pallets/system.rs

```diff
@@ -8,8 +8,26 @@ use crate::{weights::frame_system::WeightInfo, *};
 
 pub struct BaseFilter;
 impl Filter<Call> for BaseFilter {
-	fn filter(_: &Call) -> bool {
-		true
+	fn filter(call: &Call) -> bool {
+		match call {
+			// Pause Ethereum -> Darwinia bridge
+			// Around UTC 07-30-2021, 05:00 AM
+			//
+			// Due to EIP-1559, which introduces a new field in header
+			// And this break the header's scale-codec data
+			// The on-chain storage need to be migrated
+			//
+			// Ensure there is no new affirmation
+			// Then apply the migration (runtime upgrade) at UTC 07-30-2021, 10:00 AM
+			// For the started games, we expected the rounds could be finished during 05:00 AM ~ 10:00 AM
+			// TODO: remove this filter in the next runtime upgrade to restart the bridge
+			Call::EthereumRelay(darwinia_ethereum_relay::Call::affirm(..))
+				if System::block_number() > 4367266 =>
+			{
+				false
+			}
+			_ => true,
```

Comment:
@WoeOm @sxlwar wormhole might need companion PR to disable this and tell the user with the infomation.

Reply:
@AurevoirXavier I didn't get why we need to disable this, why can not just remove all existing affirmation when runtime upgrade

Reply:
Every `affirm` or `dispute_and_affirm` need to stake some fund, which uses for reward/slashing.
Once we remove the `affirmation` which will lock the fund permanently.

There is another way, remove all the storages (because they are related). Also, `take_storage_value(Stakers)` then do `Ring::remove_lock` for every staker.

Reply:
> There is another way, remove all the storages (because they are related). Also, take_storage_value(Stakers) then do Ring::remove_lock for every staker.

Let's do it this way, this will eliminate extra workloads for wormhole & bridger & community support